aten/src/ATen/native/Sorting.cpp

```diff
@@ -74,6 +74,10 @@ TORCH_META_FUNC2(sort, stable)
 (const Tensor& self, std::optional<bool> stable, int64_t dim, bool descending) {
   maybe_wrap_dim(dim, self.dim());
 
+  const auto self_dtype = self.dtype();
+  TORCH_CHECK(self_dtype != ScalarType::ComplexFloat && self_dtype != ScalarType::ComplexDouble,
```

Comment:
```suggestion
  TORCH_CHECK_VALUE(self_dtype != ScalarType::ComplexFloat && self_dtype != ScalarType::ComplexDouble,
```
Maybe be better as ValueError? Adjust test accordingly

Reply:
Yes, that makes sense, thanks for the suggestion @Skylion007. Updated accordingly.